File: Python/geometrica.py
def geometrica(n):
    if n > 1:
        return geometrica(n - 1) * 3
    return 5

def main():
    n = int(input("Ingresa hasta donde terminar la secuencia: "));
    for i in range(1, n):
	    print(geometrica(i), end = ", ")

if __name__ == '__main__':
    main();

File: C/geometrica.c
#include <stdio.h>

long int geometrica(int n) {
	if (n > 1)
		return geometrica(n - 1) * 3;
	return 5;
}

int main() {
	int n = 0;
	printf("Ingresa hasta donde terminar la secuencia: ");
	scanf("%d", &n);
	for (int i = 1; i <= n; i++)
		printf("%ld, ", geometrica(i));
}
